config/logger/logger-service.js

```diff
@@ -0,0 +1,68 @@
+import winston from 'winston';
+
+class LoggerService {
+  constructor(route) {
+    this.log_data = null;
+    this.route = route;
+
+    const logger = winston.createLogger({
+      transports: [
+        new winston.transports.Console(),
+        new winston.transports.File({
+          filename: `./logs/${route}.log`
+        })
+      ],
+      format: winston.format.printf((info) => {
+        let message = `${this.dateFormat()} | ${info.level.toUpperCase()} | ${route}.log | ${info.message} | `;
+        message = info.obj ? message + `data:${JSON.stringify(info.obj)} | ` : message;
+        message = this.log_data ? message + `log_data:${JSON.stringify(this.log_data)} | ` : message;
+        return message;
+      })
+   });
+
+   this.logger = logger;
+  }
+
+  dateFormat() {
+    return new Date(Date.now()).toUTCString();
+  }
+
+  setLogData(log_data) {
+    this.log_data = log_data;
```

Comment:
you are not following naming convention. Object property and variable have to be in camelCase

Reply:
the field was removed